ansible/inventory/env/group_vars/all.yml

```diff
@@ -204,3 +204,7 @@ content_port: 6379
 device_port: 6380
 user_port: 6381
 dialcode_port: 6382
+
+### docker image pull secrets name and this will be created in kubernetes
+imagepullsecrets: sunbird-registry-secret
```

Comment:
Please update this to be env specific. Take a look at the sunbird-devops repo for example.
`imagepullsecrets: "{{ env }}registrysecret"`

Reply:
updated as suggested.